docs/configuration-nginx.rst

```diff
@@ -0,0 +1,60 @@
+.. _nginx_configuration:
+
+Nginx configuration
+====================
+
+Apache or Nginx is not installed by default. You must choose one.
+
+Current section explains how to install and configure Nginx.
+
+Install and configure Nginx
+----------------------------
+
+On Debian:
+
+.. code-block:: shell
+
+  apt install nginx php-fpm
+
+  # configure php-fpm for Nginx
+  a2enmod proxy_fcgi setenvif
+  a2enconf php*-fpm.conf
```

Comment:
AFAIK, this is not the way to enable modules on nginx

Reply:
thx

Bad copy and paste